smellie/daqmx/config.py

```diff
@@ -2,24 +2,20 @@
 import platform
 
 if sys.platform.startswith('win'):
-    # Full path of the NIDAQmx.h file
-    # Default location on Windows XP
+    # This is the full path of the NIDAQmx.h file if it is installed on a Windows XP machine, which (as of 29/04/16) is also the default location
     dot_h_file =  r'C:\Program Files (x86)\National Instruments\NI-DAQ\DAQmx ANSI C Dev\include\NIDAQmx.h'
```

Comment:
Will do - just to be clear, this config file would still remain, and would contain only the checks for the platform and/or architecture?